src/amo/components/AddonMoreInfo/index.js

```diff
@@ -225,35 +263,24 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
   }
 
   renderDefinitions({
-    homepage = null,
-    supportUrl = null,
-    supportEmail = null,
-    statsLink = null,
-    privacyPolicyLink = null,
     eulaLink = null,
     filesize = null,
+    homepage = null,
+    privacyPolicyLink = null,
+    relatedCategories = null,
+    statsLink = null,
+    supportEmail = null,
+    supportUrl = null,
+    tagsLinks = null,
     version = null,
+    versionHistoryLink = null,
     versionLastUpdated,
     versionLicenseLink = null,
-    versionHistoryLink = null,
-    tagsLinks = null,
   }: Object): React.Node {
     const { addon, i18n } = this.props;
     return (
       <>
         <DefinitionList className="AddonMoreInfo-dl">
-          {(homepage || supportUrl || supportEmail) && (
```

Comment:
Why has this been moved?

Reply:
Ahh, you just reminded me - my initial push followed the order within the mock-up provided, but I came across a style issue I wasn't sure how to resolve when the version name is too long [(see above)](https://github.com/mozilla/addons-frontend/pull/10761#issue-688319639). Please let me know how I should proceed.

Reply:
This pull request should be purely related to exposing categories in more info - if there are problems with version names, or the metadata order being slightly different to the original mocks, or anything else they should be dealt with in another issue.